clang/tools/clang-format/git-clang-format

```diff
@@ -126,6 +126,7 @@ def main():
             "pb.txt",
             "textproto",
             "asciipb",  # TextProto
+            "cl", # OpenCL
```

Comment:
Move this line to just below line 108 above? Do we want to add `"clcpp", # OpenCL C++`? If yes, we should add it to clang-format-diff.py too.

Reply:
done, moved after line 108. 
> Do we want to add "clcpp", # OpenCL C++?

I actually don't know about the current development status of C++ for OpenCL in clang. So If there is such request, I think it can be added in a separate PR.